tools/global_setup.bat

```diff
@@ -0,0 +1,51 @@
+@echo off
+ECHO "Starting setup..."
+
+:ontology_validator_setup
+ECHO "Starting ontology validator setup"
+CD validators\ontology_validator
+START py setup.py install
```

Comment:
Using "START" causes all of the tools to install simultaneously in their own terminal, which causes failures for the components that depend on the ontology validator.

You can remove "START" from the start of these lines to have the setups run in sequence, and maybe add the "-q" quiet option to reduce output from the installation process.

Reply:
Ok I will remove START and give that a shot.

Reply:
START is still there in the latest commit.